apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/ServiceOptGroup.tsx

```diff
@@ -67,7 +68,19 @@ const ServiceOptGroup = ({
         }
 
         if (service.id === todayServiceId) {
-          optionText += " (now)";
+          // if it's the only service this rating, this service might not
+          // technically be now - adjust text for that
+          const now = Date.now();
+          if (isInCurrentService(service, new Date(now))) {
```

Comment:
nit-pick: Since I don't see now being used anywhere else this could just be `new Date()`

Reply:
true, though it made it easier to mock the date in Jest, via 
```javascript
jest
      .spyOn(Date, "now")
      .mockImplementation(() => new Date("2019-09-07").getTime());
```

do you know if there's a straightforward way to mock `new Date()`?

Reply:
Oh, I see your conundrum.  What we have been doing else where is make it an optional prop/param that defaults to `new Date()`.  Then you can pass the desired date into the prop for testing.